analysis/metrics.py

```diff
@@ -59,9 +59,16 @@ class SumMetrics:
     noise_kind: pipeline_dp.NoiseKind
 
 
+@dataclass
+class Statistics:
```

Comment:
I don't think this is very descriptive. Maybe something like `RawStatistics` could be more appropriate?

Reply:
Thanks, it's a good point. I've renamed to RawStatistics